integration/run_integration_test.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 #
-# run_integration_test.sh is a wrapper around the log_prep and run_test functions.
-# It's intended to be used by the Travis CT/Trillian integration test only,
-# and will go away when we migrate off of Travis.
+# run_integration_test.sh is a wrapper around the log_prep and run_test
+# functions. It's intended to be used by the Travis CT/Trillian integration test
+# only, and will go away when we migrate off of Travis.
+#
+# TODO(al): Make the script go away.
```

Comment:
@AlCutter Based on "blame" for this file, you wrote this. I am not sure how this file is used.

Reply:
I think it was a workaround for something. Al will know. It's referenced from travis.yaml so you might be removing the only usage.

Reply:
Ah, I think there was some test using this: It cloned CT repo and ran this script from within CT.

We don't have such test in Cloud Build. Should we replicate it? It seems that such a thing should be tested in CT repo rather than Trillian?